src/git-data.ts

```diff
@@ -82,7 +82,7 @@ export class GitData {
         } else {
             throw new ExitError("Could not locate.gitconfig or .git/config file");
         }
-        const gitRemoteMatch = gitConfig.match(/url = .*(?:http[s]?:\/\/|@)(?<domain>.*?)[:|/](?<group>.*)\/(?<project>.*?)(?:\r?\n|\.git)/);
+        const gitRemoteMatch = gitConfig.match(/url = .*(?:http[s]?:\/\/|@)(?<domain>.*)[:|/](?<group>.*)\/(?<project>.*?)(?:\r?\n|\.git)/);
```

Comment:
i am not sure this regex changes fixes all cases - i would go with something like 
```
/.*(?:http[s]?:\/\/|@)(?<domain>[^:/]*)((:\d+\/)|:|\/){1}(?<group>.*)\/(?<project>.*?)(?:\r?\n|\.git)/
```

i adapted also the current test to be better understandable, so if you rebase onto the latest main and adapt the the `git remote only test` to something like the following - this verifies against a lot of combinations and can be easily adapted

```javascript
   describe("only for git remote data", () => {
        describe("for working input", () => {
            test.each([
                `git@${remote.domain}:${remote.group}/${remote.project}.git`,
                `git@${remote.domain}/${remote.group}/${remote.project}.git`,
                `git@${remote.domain}:222/${remote.group}/${remote.project}.git`,
                `https://git:git@${remote.domain}/${remote.group}/${remote.project}.git`,
                `https://git:git@${remote.domain}:22/${remote.group}/${remote.project}.git`,
            ])("%s", async (url) => {
                when(mock)
                    .calledWith("git remote -v", expect.any(String))
                    .mockReturnValue({
                        stdout: `
                    origin	${url} (fetch)
                    origin	${url} (push)
                    `
                    })
                Utils.spawn = mock
                const gitData = await GitData.getRemoteData("./");
                expect(gitData).toEqual(remote)
            })
        })
```